language/Turkish/strings.po

```diff
@@ -4345,12 +4368,12 @@ msgstr ""
 #: src/emu/inpttype.ipp:115
 msgctxt "input-name"
 msgid "High"
-msgstr ""
+msgstr "Yüksek"
 
 #: src/emu/inpttype.ipp:116
 msgctxt "input-name"
 msgid "Low"
-msgstr ""
+msgstr "Düşük"
```

Comment:
never played gambling games in Mame (and real life) .
"3 is higher than 2" = "3 2'den yüksek"
"2 is lower than 3"  = "2 3'ten düşük"
in this case translation is correct.
We can also call "3 2'den büyüktür" = "3 is bigger than 2".
I can test it later with gambling games(i also have to test all translations because sometimes even the translations are correct they don't fit the used situation)